admin-ui/app/locales/pt/translation.json

```diff
@@ -508,6 +508,7 @@
     "scope_type": "Tipo de Escopo",
     "scopes:": "Scopes",
     "scripts": "Scripts",
+    "ldap_servers": "Servidores LDAP",
     "script_type": "Tipo de Script",
```

Comment:
Key placed under wrong namespace: move ldap_servers to menus

EN uses menus.ldap_servers. PT added fields.ldap_servers, which will break lookups.


```diff
-    "ldap_servers": "Servidores LDAP",
```

Add under the "menus" section (near other menu labels):

```diff
@@ "menus": {
     "scripts": "Roteiros",
+    "ldap_servers": "Servidores LDAP",
     "services": "Serviços",
```